### PR TITLE
Add resource policies doc

### PR DESCRIPTION
**Pull request by qiuming-best**
Thank you for contributing to Velero!

# Please add a summary of your change

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [ ] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Review by sseago — DISMISSED** [review]

File: site/content/docs/main/resource-filtering.md [hunk 1]
```diff
@@ -201,3 +203,121 @@ Kubernetes namespace resources to exclude from the backup, formatted as resource
   ```bash
   velero backup create <backup-name> --exclude-namespaced-resources="*"
   ```
+
+## Resource policies
+Velero provides resource policies to filter resources to do backup or restore. currently, it only supports skip backup volume by resource policies.
+
+**Creating resource policies**
+
+below is the two-step of using resource policies to skip backup of volume:
```
Comment (ywk253100) [inline site/content/docs/main/resource-filtering.md:212]: below -> Below

Comment (qiuming-best) [reply site/content/docs/main/resource-filtering.md]: thanks, updated

**Review by ywk253100 — DISMISSED** [review]

File: site/content/docs/main/resource-filtering.md [hunk 2]
```diff
@@ -201,3 +203,121 @@ Kubernetes namespace resources to exclude from the backup, formatted as resource
...
+        csi: {}
+      action:
+        type: skip
+    ```
+
+**Supported conditions**
+
+Currently, Velero supports the volume attributes listed below:
+- capacity: matching volumes have the capacity that falls within this `capacity` range. The capacity value should include the lower value and upper value concatenated by commas. And it has several combinations below:
+  - "0,5Gi" or "0Gi,5Gi" which means capacity or size matches from 0 to 5Gi, including value 0 and value 5Gi
```
Comment (blackpiglet) [inline site/content/docs/main/resource-filtering.md:280]: Not sure about the "0,5Gi" setting.
It looks like "0Gi,5Gi" is more reasonable.
In my testing, "10,100Gi" is not the same as "10Gi,100Gi".

Comment (qiuming-best) [reply site/content/docs/main/resource-filtering.md:280]: Maybe I should make a clearer explanation in the doc, the Unit of the capacity could be `Gi`, `Mi`, `Ki` etc.  
The Unit of the Capacity is the standard K8S storage unit. so "10,100Gi" is not equal with "10Gi,100Gi"

Comment (qiuming-best) [reply site/content/docs/main/resource-filtering.md:280]: updated

**Review by ywk253100 — APPROVED** [review]

**Pull request merged**